package-update/overlays/stage/imagestreamtag.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    app: thoth
+    component: package-update
+  name: package-update-job
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/thoth-station/package-update-job:v0.6.0-dev
+      importPolicy: {}
+      referencePolicy:
+        type: Source
```

Comment:
a) is this still a valid option on ocp4.3 ? and b) should we reconfigure stage to use `local` to reduce coupling with quay?

Reply:
thanks for pointing it.